lib/phone_shop/widgets/components/overview_card.dart

```diff
@@ -0,0 +1,203 @@
+import 'package:flutter/material.dart';
+
+class OverviewCard extends StatelessWidget {
+  final String title;
+  final String category;
+  final double discountPercentage;
+  final double price;
+  final double rating;
+  final String description;
+
+  const OverviewCard({
+    super.key,
+    required this.title,
+    required this.category,
+    required this.discountPercentage,
+    required this.price,
+    required this.rating,
+    required this.description,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      color: const Color(0xFFFFFFFF),
+      shape: RoundedRectangleBorder(
+        borderRadius: BorderRadius.circular(12),
+        side: const BorderSide(
+          color: Color(0xFFF1F1F1),
+          width: 1,
+        ),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.all(16),
+        child: Column(
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                _Category(category: category),
+                _Price(price: price),
+              ],
+            ),
+            const SizedBox(height: 8),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                _Title(title: title),
+                _Discount(discountPercentage: discountPercentage),
```

Comment:
I want Brand and Title to brake in more lanes if card is too small to contain it in one lane. Never leave overflow.